lib/config.js

```diff
@@ -152,6 +153,22 @@ const conf = convict({
       default: 10
     }
   },
+  openid: {
+    key: {
+      doc: 'Private JWK to sign id_tokens',
+      default: {}
+    },
```

Comment:
We should get ops/opsec input on the right way to load this key from a secret-management perspective, e.g. in the auth-server we load it from a separate file, which might make some things with permissions easier.